lib/platform/bitbucket-server/utils.ts

```diff
@@ -118,3 +122,38 @@ export interface BitbucketStatus {
   key: string;
   state: BitbucketBranchState;
 }
+
+interface BitbucketErrorResponse {
+  errors?: Array<{
+    exceptionName?: string;
+    reviewerErrors?: Array<{ context?: string }>;
+  }>;
```

Comment:
```suggestion
  errors?: {
    exceptionName?: string;
    reviewerErrors?: { context?: string }[];
  }[];
```
uses short array type to be more consistent

Reply:
👍